hc/front/tests/test_period_update_check.py

```diff
@@ -0,0 +1,22 @@
+from hc.api.models import Check
+from hc.test import BaseTestCase
+
+
```

Comment:
One line space is enough

Reply:
hey @jimmykimani from your comment should we not leave two lines blank lines between modular level declarations and class definitions as per PEP8 standards?